lib/ansible/modules/cloud/ovirt/ovirt_host.py

```diff
@@ -75,6 +75,7 @@
         description:
             - "Indicates that the host should be removed even if it is non-responsive,
                or if it is part of a Gluster Storage cluster and has volume bricks on it."
+            - "It don't forcibly remove host if it's being removed already."
```

Comment:
I'd rephrase:
 "WARNING: It doesn't forcibly remove the host if another host related operation is being executed on the host at the same time."